UnitTests/UnitTests.HighPerformance.UWP/UnitTests.HighPerformance.UWP.ruleset

```diff
@@ -0,0 +1,236 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Microsoft Managed Recommended Rules" Description="These rules focus on the most critical problems in your code, including potential security holes, application crashes, and other important logic and design errors. It is recommended to include this rule set in any custom rule set you create for your projects." ToolsVersion="14.0">
+  <Localization ResourceAssembly="Microsoft.VisualStudio.CodeAnalysis.RuleSets.Strings.dll" ResourceBaseName="Microsoft.VisualStudio.CodeAnalysis.RuleSets.Strings.Localized">
+    <Name Resource="MinimumRecommendedRules_Name" />
+    <Description Resource="MinimumRecommendedRules_Description" />
+  </Localization>
+  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
```

Comment:
Not sure, that wasn't intentional on my part, I just went with setup that VS generated automatically when I created the UWP unit test project for the package. Looks like the unit test project for the main toolkit package also has a very similar `.ruleset` file ([see here](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/UnitTests/UnitTests.ruleset)) for some reason.  Should I just delete this `.ruleset` file from my unit test project then, since I don't need that in particular?

Reply:
For future reference, resolved in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3128/commits/abf4bebf6dac5e0376ec5dc9792b672e9294ff46.